src/cli.ts

```diff
@@ -296,7 +296,7 @@ async function openPage(context: playwright.BrowserContext, url: string | undefi
 
 type Language = 'python' | 'python-async' | 'javascript' | 'csharp';
 
-async function open(options: Options, url: string | undefined, enableRecorder: boolean, language?: Language, outputFile?: string) {
+async function open(options: Options, url: string | undefined, enableRecorder: boolean, language: Language = 'javascript', outputFile?: string) {
```

Comment:
I think the right fix would be to group recorder-related parameters (`enableRecorder`, `language`, `outputFile`, `terminalLanguage`, `languageGenerator`) into a struct, default that struct to undefined, and only initialize it in `codegen()`.

Otherwise, we have to workaround stuff like this, although `open` does not care about language per se.

WDYT?